### DISCUSSION TITLE
How to move a menu item to the bottom of the menu separated from the rest of the menu items without using <Menu.Item>

### DISCUSSION THREAD

**Discussion by GeorgeFlorian** [ant-design/ant-design#36430] [General]
### What problem does this feature solve?

I would like to take this example: https://codesandbox.io/s/pf1pq9?file=/demo.js and move the `Files` item to the bottom of the menu, above the `sider` arrow/button.

A solution to this problem would make styling individual menu items easier while using the new `<Menu items={[...]} />` .

Right now I can't find any way of styling individual menu items while using `<Menu items={[...]} />` , except maybe manipulating tons of CSS which defeats the purpose of using this library.

**Comment by vonPB** [comment]
Hey @GeorgeFlorian 
have you found an solution for this?
